projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-input/search-ai-input.component.ts

```diff
@@ -118,13 +118,14 @@ export class SearchAiInputComponent implements OnInit, OnDestroy {
     } else {
       this.selectedNodesState = JSON.parse(this.userPreferencesService.get(this.storedNodesKey));
     }
+
     this.agentService
       .getAgents()
       .pipe(takeUntil(this.onDestroy$))
       .subscribe(
-        (paging) => {
-          this._agents = paging.list.entries.map((agentEntry) => agentEntry.entry);
-          this.agentControl.setValue(this.agents.find((agent) => agent.id === this.agentId));
+        (agents: AgentWithAvatar[]) => {
```

Comment:
similarly like for agents button I think no need to have type here because it should be autodetected based on getAgents results.